src/generate.ts

```diff
@@ -112,17 +112,18 @@ export async function generateApi(
     });
   }
 
-  const baseQueryCall = factory.createCallExpression(factory.createIdentifier(baseQuery), undefined, [
-    factory.createObjectLiteralExpression(
-      [
-        factory.createPropertyAssignment(
-          factory.createIdentifier('baseUrl'),
-          factory.createStringLiteral(baseUrl as string)
-        ),
-      ],
-      false
-    ),
-  ]);
+  const generateBaseQueryCall = () =>
+    factory.createCallExpression(factory.createIdentifier(baseQuery), undefined, [
+      factory.createObjectLiteralExpression(
+        [
+          factory.createPropertyAssignment(
+            factory.createIdentifier('baseUrl'),
+            factory.createStringLiteral(baseUrl as string)
+          ),
+        ],
+        false
+      ),
+    ]);
```

Comment:
```suggestion
  const fetchBaseQueryCall = factory.createCallExpression(factory.createIdentifier('fetchBaseQueryCall'), undefined, [
    factory.createObjectLiteralExpression(
      [factory.createPropertyAssignment(factory.createIdentifier('baseUrl'), factory.createStringLiteral(baseUrl))],
      false
    ),
  ]);
```

I think we can keep this just as it is and don't need a function - this will just create an object. It doesn't hurt if it never used later.

Let's just remove the `as string` cast as that doesn't seem necessary any more now.

Also let's make it specific to `fetchBaseQuery` and give it an according name